### PR TITLE
added PartiesPositionPreferences back into client

### PR DESCRIPTION
**Pull request by xthesebx**

### CODE REVIEW

**Comment by xthesebx** [comment]
failed commit message for last one, "read" is just fixing invites and joining lobbies cause im stupid

**Comment by xthesebx** [comment]
now also has the presence stuff in it
can you pls fix the merge conflicts, i dont know how exactly you want this stuff especially with the champselect

File: Core/src/main/java/com/hawolt/client/misc/MapQueueId.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	}
+	
+	public static String getMapId(int queueId) {
+		switch (queueId) {
+			case 400, 430, 420, 440, 830, 840, 850: return "11";
+			case 450: return "12";
+			case 1090, 1100, 1130, 1160: return "22";
+		}
+		return "11";
+	}
```
Comment (JanSuran03) [inline Core/src/main/java/com/hawolt/client/misc/MapQueueId.java:37]: Replace with enhanced switches, add the default return to the switch default branch.

Comment (hawolt) [reply Core/src/main/java/com/hawolt/client/misc/MapQueueId.java:37]: done :)

**Pull request merged**